app/geodjango/settings.py

```diff
@@ -124,7 +130,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "America/Chicago"
```

Comment:
Just want a gut check on this. I don't think we currently have anything more granular than day-level datetimes so this shouldn't cause huge problems, but if we were to have any daily schedule data or if we had users submit the timeframe they want to take a particular route we will have to translate to Chicago time. 

Usually when there is conversion I have seen everything get converted to UTC so that we know it is all fully standardized, but if you are ok with keeping Chicago then that is fine, we will just have to note that in future hourly data for non-chicago places will need to be translated.